PowerEditor/src/Parameters.h

```diff
@@ -46,6 +46,7 @@
 
 #endif
 
+#define CMD_INTERPRETER TEXT("%WINDIR%\\System32\\cmd.exe")

```

Comment:
Better use constants in C++, instead of defines ?

`const TCHAR cmdInterpreter[] = TEXT("%WINDIR%\\System32\\cmd.exe");`

Reply:
It is equivalent (the above `#define...` will become the same `const TCHAR*`).